### DISCUSSION TITLE
Another channel search issue

### DISCUSSION THREAD

**Discussion by aliessum** [Yogui79/IntexPureSpa#52] [Q&A]
_I'm new to Github, hopefully my post is in the right location._

First of course many thanks for the effort you have put in the design!
I am facing the same problem as described in https://github.com/Yogui79/IntexPureSpa/discussions/20.
My spa is a Intex type #2845800A, according to the carton box. I have a LC12S V2.03 module (so not the blue one, but red/light brown like on your picture). ESP32 is a devkit C V4, so the one with the extending antenna board.
During channel search the blue led is flashing slow and in the console I can see the numbers counting up. Distance to the pump-unit is less than 1 mtr, but a channel is not detected.
Is there a way that I can check the functionality of the LC12S (perhaps with an oscilloscope, I have one)?
Have you had experiences with bad LC12S modules, should I maybe get another LC12S?
Thanks in advance, Arno.

**Comment by stevenbrown1862** [comment]
Have you tried doing it in the dark?  I couldn't tell it was flashing fast unless the room was dark.  I also increased one of the times in the sketch which I thought would make it flash fast for longer but I can't remember which one.   Doing it in the dark was the game changer though.

**Reply by aliessum** [reply]
Hi Steven, thanks for your reply. I would have expected the scanner to stop at the designated channel and flash fast. Reading your answer it seems not the case? Ok I will try again and see what happens. Thanks again.

**Comment by stevenbrown1862** [comment]
Yes it definitely carries on.  From memory it flashed on 2 consecutive
channels for mine.   Invisible in the light but really clear in the dark.

**Reply by aliessum** [reply]
Hi all,
I changed the scan-time to 5000 ms and tried this evening (distance approx 0,5 mtr). I could now clearly see the blue led fast flashing at 2 consecutive channels: 41 and 42 (hex), brightness identical. Happy so far that the electronics is ok and works with our type of spa!! But for which channel should I go, or just try in the next step?

**Comment by stevenbrown1862** [comment]
I think I just picked the first one.  No idea whether that was right or not.

**Reply by aliessum** [reply]
Hi all,
Did an experiment, now I know that the channel is in the 40-range, I decreased the loop to channel 40-48 (scan time 5000 ms). This saves some waiting time...
I  also increased the distance to about 5 mtr and position in our house. Still got a firm response (and visible with daylight), but now only on channel 41 (hex), so that must be the one.

**Reply by aliessum** [reply]
Scan for NetworkId works perfect as well. After installing the Spa.ino I could see data from from the spa in my MQTT Explorer. Also sending a command work immediately! So problem solved!

**Reply by dwavrant** [reply]
hello @aliessum 

Could you tell me if you had a rapid blinking  during the networkIf search with the correct channel id ?
like this video : https://github.com/Yogui79/IntexPureSpa/blob/main/Docs/Video/RightChannel.mp4

I think I identified some channel id (4), but I never have a rapid blinking during the network id searching :(

TY

**Reply by aliessum** [reply]
Hi,
During scanning for the correct network id, the led should blink rapidly all the time. So, if not, you have the wrong channel id or your spa is not supported.
Did you try to extend the scan time to 5000 ms? This helps identifying the right channel.

Kind regards,
Arno

**Reply by dwavrant** [reply]
Hum, I'm afraid my spa is not supported :(
Yes, I extend the scan to 5000ms for the channel and did it in the dark. I found 4 potential channels, but no one with a rapid blink on networkid search
Finally, I modified the netword id search program to test 3 network id on each channel (from 00 to FF), and no channel produces rapid blink :(

**Comment by LEECHER1** [comment]
Nice :)

Thanks for the Info.
Which intex spa do you have?

Br

Arno ***@***.***> schrieb am Di., 5. Juli 2022, 20:50:

**Reply by aliessum** [reply]
Hi Leecher1, 
We have the Intex 28458.
Regards, Arno
